css-four-column-layout/index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="UTF-8">
+  <title>css-four-column-layout</title>
+  <link rel="stylesheet" href="style.css">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Barlow:wght@400;800&display=swap" rel="stylesheet">
+</head>
+<body>
+  <div class="container">
+    <h2>How it Works</h2>
+    <div class="row">
+      <div class="column-full">
```

Comment:
This `column-full` isn't needed, you can just have the four `column-fourth` elements in the `row` as children

Reply:
👌🏻